src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -42,25 +42,20 @@
     * Create an {@service.clientName} with additional helpers for common
     * tasks.
     *
-    * @@param {Object=} opts - The optional parameters.
-    * @@param {String=} opts.servicePath
-    *   The domain name of the API remote host.
-    * @@param {number=} opts.port
-    *   The port on which to connect to the remote host.
-    * @@param {grpc.ClientCredentials=} opts.sslCreds
-    *   A ClientCredentials for use with an SSL-enabled channel.
-    * @@param {Object=} opts.clientConfig
-    *   The customized config to build the call settings. See
-    *   {@@link gax.constructSettings} for the format.
+    {@comments(@service.doc.lines)}
+    *
+    * @@param {Object=} options - The optional parameters.
```

Comment:
Can we also lowercase the types (i.e. `Object`, `String`) since you're already in here?